CommonTools/ParticleFlow/plugins/PFPileUp.cc

```diff
@@ -192,4 +200,20 @@ void PFPileUp::produce(Event& iEvent, const EventSetup& iSetup) {
   iEvent.put(std::move(pOutput));
   // iEvent.put(std::move(pOutputByValue));
 }
+
+void PFPileUp::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("PFCandidates", edm::InputTag("particleFlowTmpPtrs"));
+  desc.add<edm::InputTag>("Vertices", edm::InputTag("offlinePrimaryVertices"));
+  desc.add<bool>("Enable", true);
+  desc.addUntracked<bool>("verbose", false);
+  desc.add<bool>("checkClosestZVertex", true);
```

Comment:
Having this in the fillDescription allows removing the check `iConfig.exists("checkClosestZVertex")` at L105
Please notice that now the defauld for this parameter in case it is missing from the configuration is `true`, while it was `false` before having the `fillDescriptions`